playbooks/utilities/mofed.yml

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
Is it necessary to add a few lines of documentation to the top of this playbook or do you think it is self-describing?

Reply:
Documentation is always nice to have, I can add some.